aeppl/transforms.py

```diff
@@ -237,11 +299,27 @@ def log_jac_det(self, value, *inputs):
         return at.zeros(value.shape)
 
 
-def create_default_transformed_rv_op(
+def _create_transformed_rv_op(
```

Comment:
This method is going to be further expanded in #26 so it seems reasonable to make it private and just expose the user facing `create_default_transformed_rv_op` with the fixed default kwargs (via the partial)

Reply:
This change doesn't make the function private in any functional sense; it's just a naming convention.  If we're concerned about the things we're exposing to users, the best we can do is set `__all__`.

Regardless, this is a minor style-based refactoring that should be in a separate PR/commit.